src/components/Container/Container.js

```diff
@@ -24,10 +30,16 @@ const Container = props => {
         <MovieLoader key={1}/>,
         <MovieLoader key={2}/>,
         <MovieLoader key={4}/>]
```

Comment:
I think we can clean up the spacing on this array by separating the elements from the brackets.